ard/collection/optiwindnet_wrap.py

```diff
@@ -22,13 +25,37 @@ def _own_L_from_inputs(inputs: dict, discrete_inputs: dict) -> nx.Graph:
     VertexC[:T, 1] = inputs["y_turbines"]
     VertexC[-R:, 0] = inputs["x_substations"]
     VertexC[-R:, 1] = inputs["y_substations"]
+
+    # add jitter to duplicate turbine/substation positions
+    VertexCTR = np.vstack([VertexC[:T, :], VertexC[-R:, :]])
+    jitter_eps = 1.0e-3  # 0.001m magnitude jitter
+    jitter_normal = np.array([-1.0, 1.0])  # on a fixed axis we'll normalize
+    jitter_normal = jitter_normal / np.sqrt(np.sum(jitter_normal**2))
+    repeat_accumulate = np.array(
+        [
+            int(np.sum(np.all(VertexCTR[:ivv, :] == vv, axis=1)))
+            for ivv, vv in enumerate(VertexCTR)
+        ]
+    )
+    if np.any(repeat_accumulate > 0):
+        warn(
+            f"WARNING: detected {np.sum(repeat_accumulate > 0)} coincident "
+            f"turbines and/or substations in optiwindnet setup..."
```

Comment:
I think it would be good to include a description in the warning message of the automatic adjustment being made to the vertices, maybe even including a list of adjusted vertices.